recipe/build.sh

```diff
@@ -21,11 +21,17 @@ cp ${MARIADB_VERSION}/COPYING .
 mv tmp ${MARIADB_VERSION}/storage/mytile
 cd ${MARIADB_VERSION}
 
-# tools
-mkdir host
-cd host
-cmake ..
-make import_executables
+
+export CC=${CC_FOR_BUILD} \
```

Comment:
This needs to be conditional on `target_platform` (see below)

Reply:
(otherwise it breaks the non-cross build, see linux failures)

Reply:
Yes I know just tried something quick..